api/BlobBuilder.json

```diff
@@ -44,7 +44,8 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": null
+            "version_added": true,
```

Comment:
Check this is only available via prefix, I think we can leave off the prefix.

Reply:
We do not support this
![image](https://user-images.githubusercontent.com/4225330/67209276-fb0a8180-f40e-11e9-99a8-5c8e7cc5e1e1.png)